go/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/google/flatbuffers/go
```

Comment:
FYI adding this go.mod will probably break things.
https://github.com/google/flatbuffers/issues/7780